src/apollo/client.ts

```diff
@@ -185,10 +226,11 @@ const getTypePolicies = (autologin: boolean): TypedTypePolicies => ({
 
 export const createApolloClient = (
   apiUrl: string,
-  autologin: boolean
+  autologin: boolean,
+  fetchOptions?: FetchConfig
```

Comment:
Tests test different time skew of token refresh, thus it needs to explicitly define them. Also if we are able to change time skew for token refresh in custom fetch created by `createFetch` method, we should consequently allow to set the same time skew for token refresh for SDK `createSaleorClient`.
And if we want to answer the question if we would allow to change time skew - I guess we should as user may modify token expiration time on backend and in this case SDK should allow to modify refresh skew time also.